src/components/xp/part.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+  import type {Content} from "enonic-types/content";
+  import type {Parts} from "$components/xp/types";
+  import type {Component} from "enonic-types/portal";
+
+  export let content: Content;
+  export let component: Component<any>;
```

Comment:
Try to avoid using `any`. Many times you can use `unknown`, which is a safer choice instead.

```suggestion
  export let component: Component<unknown>;
```